src/main/java/org/jenkinsci/plugins/azurekeyvaultplugin/AzureCredentialsProvider.java

```diff
@@ -107,9 +108,14 @@ static String getSecretName(String itemId) {
             PagedIterable<SecretProperties> secretItems = client.listPropertiesOfSecrets();
             for (SecretProperties secretItem : secretItems) {
                 String id = secretItem.getId();
-                SecretStringCredentials cred = new SecretStringCredentials(CredentialsScope.GLOBAL, getSecretName(id),
-                        id, credentialID, id);
-                credentials.add(cred);
+                Map<String, String> tags = secretItem.getTags();
```

Comment:
this seems to always return null for me

need to do a bit more digging, this issue was related to 'keys' maybe there's a similar issue for secrets

Reply:
For me this works fine using `com.azure:azure-security-keyvault-secrets:3.1.0` (+ `com.microsoft.azure:azure-keyvault:1.0.0` + `com.microsoft.azure:azure-keyvault-core:0.8.0`), what versions do you get this issue with?

Reply:
dumb me I didn't hit save in the UI in the portal, I pushed a fix for secrets without tags

Reply:
Good catch, thanks!